tuf-spec.md

```diff
@@ -1586,12 +1586,12 @@ We now explain how a repository should write metadata and targets to
 produce self-contained consistent snapshots.
 
 Simply put, TUF should write every metadata file as such: if the
-file had the original name of filename.ext, then it should be written to
-non-volatile storage as version_number.filename.ext, where version_number
-is an integer.
+file had the original name of FILENAME.EXT, then it should be written to
+non-volatile storage as VERSION_NUMBER.FILENAME.EXT, where VERSION_NUMBER
+is the integer version number listed in the metadata file.
 
 On the other hand, consistent target files should be written to
-non-volatile storage as digest.filename.ext.  This means that if the
+non-volatile storage as HASH.FILENAME.EXT.  This means that if the
```

Comment:
In general I like `DIGEST` better than `HASH`, but I know that we use `HASH` elsewhere in the spec. No need to change it unless others feel strongly.

Reply:
I used `HASH` because that's what we use in the the [5.7.3](https://theupdateframework.github.io/specification/v1.0.25/#fetch-target) of the detailed client workflow and the name of the attribute in metadata is 'hashes'.